main.js

```diff
@@ -1,5 +1,5 @@
 /* SETUP */
-var map = L.map('map', {
+let map = L.map('map', {
```

Comment:
we should leave this as `var` as it won't work in some standard browsers that don't support es6

Reply:
probably not many there but still, lets just keep it in vanilla js